test/test_prototype_transforms.py

```diff
@@ -188,3 +189,58 @@ def test_random_resized_crop(self, transform, input):
     )
     def test_convert_image_color_space(self, transform, input):
         transform(input)
+
+
+@pytest.mark.parametrize("p", [0.0, 1.0])
+class TestRandomHorizontalFlip:
+    def input_expected_image_tensor(self, p, dtype=torch.float32):
+        input = torch.tensor([[[0, 1], [0, 1]], [[1, 0], [1, 0]]], dtype=dtype)
+        expected = torch.tensor([[[1, 0], [1, 0]], [[0, 1], [0, 1]]], dtype=dtype)
+
+        if p == 1.0:
+            return input, expected
+        return input, input
```

Comment:
```suggestion
        return input, expected if p == 1 else input
```

Reply:
No problem! I saw you already fixed :)

Thank you for the reviews!